packages/cli/src/databases/repositories/execution.repository.ts

```diff
@@ -110,13 +118,17 @@ export class ExecutionRepository extends Repository<ExecutionEntity> {
 	}
 
 	setSoftDeletionInterval() {
-		this.logger.debug('Setting soft-deletion interval (pruning) for executions');
+		this.logger.debug(
+			`Setting soft-deletion interval (pruning) for executions with rate ${this.rates.hardDeletion}`,
+		);
 
 		this.intervals.softDeletion = setInterval(async () => this.prune(), this.rates.hardDeletion);
 	}
 
 	setHardDeletionInterval() {
-		this.logger.debug('Setting hard-deletion interval for executions');
+		this.logger.debug(
+			`Setting hard-deletion interval for executions with rate ${this.rates.hardDeletion}`,
```

Comment:
These are ms - for display we might want to transform and add the unit. Same for the other.